stream-chat-android-ui-components/src/main/res/layout/stream_message_list_view.xml

```diff
@@ -101,32 +101,253 @@
     </FrameLayout>
 
     <FrameLayout
-        android:id="@+id/blurLayer"
+        android:id="@+id/shadowLayer"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:visibility="gone"
         android:background="#C000"
+        android:visibility="gone"
         />
 
     <ScrollView
         android:id="@+id/messageOptionsScroll"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:clickable="false"
-        android:visibility="visible"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         >
 
-        <FrameLayout
-            android:id="@+id/messageOptionsContainer"
+        <LinearLayout
+            android:id="@+id/optionRootContainer"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:clickable="false"
-            android:minHeight="200dp"
-            />
+            android:orientation="vertical"
+            android:visibility="gone"
+            tools:visibility="visible"
+            >
+
+            <FrameLayout
+                android:id="@+id/optionsMessageContainer"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:clickable="false"
+                />
+
+            <androidx.cardview.widget.CardView
+                android:id="@+id/optionListContainer"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="50dp"
+                android:layout_marginTop="8dp"
+                android:layout_marginEnd="50dp"
+                android:layout_marginBottom="8dp"
+                android:padding="8dp"
+                app:cardCornerRadius="12dp"
+                >
+
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:minWidth="200dp"
+                    android:paddingTop="12dp"
+                    android:paddingBottom="12dp"
+                    >
+
+                    <ImageView
```

Comment:
A bit of duplication here .Can be optimized later with LinearLayout with dividers and TextViews with `drawableLeft`